.github/workflows/ci_linux.yml

```diff
@@ -8,7 +8,7 @@ on:
   pull_request:
 
 env:
-  CMAKE_VERSION: 3.11.4
+  CMAKE_VERSION: 3.16.0
```

Comment:
This should be 3.16.9 (latest patch level). Googletest will require at least 3.16.3 when updated